packages/typescript-estree/package.json

```diff
@@ -43,6 +43,6 @@
   },
   "devDependencies": {
     "@typescript-eslint/shared-fixtures": "1.2.0",
-    "typescript": "~3.2.1"
+    "typescript": "~3.3.1"
```

Comment:
FYI, we unfortunately have to have this referenced in the package.json of typescript-estree, because it is used as the source of truth for the supported version, and that is something needed at runtime, when the package is no longer in the context of the monorepo